src/container/app.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component} from 'react'
+import Map from '../component/map'
+import {TextField, List, ListItem} from '@material-ui/core'
+import { withStyles } from '@material-ui/core/styles'
+import PlacesAutocomplete, {
+  geocodeByAddress,
+  getLatLng, loading
+} from 'react-places-autocomplete'
+import {connect} from 'react-redux'
```

Comment:
Be sure to have consistent whitespace in your named imports and destructuring.

`{connect}`
`{ withStyles }`